selfdrive/car/gm/values.py

```diff
@@ -53,6 +58,8 @@ class CAR:
   ACADIA = "GMC ACADIA DENALI 2018"
   BUICK_REGAL = "BUICK REGAL ESSENCE 2018"
   ESCALADE_ESV = "CADILLAC ESCALADE ESV 2016"
+  SILVERADO = "CHEVROLET SILVERADO 1500 2020"
```

Comment:
Note: Apparently the "1500" is part of the name: https://media.chevrolet.com/media/us/en/chevrolet/vehicles/silverado/2020.html